diff --git a/article_netmgr.cc b/article_netmgr.cc
index 00f3105..414f64c 100644
--- a/article_netmgr.cc
+++ b/article_netmgr.cc
@@ -45,7 +45,7 @@ using std::string;
 
       for( QSet< QPair< QString, QString > >::iterator it = hostsToAccess.begin();
            it != hostsToAccess.end(); ++it )
-        origin->addAccessWhitelistEntry( it->first, it->second, QWebSecurityOrigin::AllowSubdomains );
+        origin->addAccessAllowlistEntry( it->first, it->second, QWebSecurityOrigin::AllowSubdomains );
     }
   }
 
@@ -55,7 +55,7 @@ using std::string;
     {
       for( QSet< QPair< QString, QString > >::iterator it = hostsToAccess.begin();
            it != hostsToAccess.end(); ++it )
-        origin->removeAccessWhitelistEntry( it->first, it->second, QWebSecurityOrigin::AllowSubdomains );
+        origin->removeAccessAllowlistEntry( it->first, it->second, QWebSecurityOrigin::AllowSubdomains );
 
       delete origin;
       origin = 0;
@@ -295,7 +295,7 @@ QNetworkReply * ArticleNetworkAccessManager::createRequest( Operation op,
           if( value.hostsToAccess.find( target ) == value.hostsToAccess.end() )
           {
             value.hostsToAccess.insert( target );
-            value.origin->addAccessWhitelistEntry( target.first, target.second,
+            value.origin->addAccessAllowlistEntry( target.first, target.second,
                                                    QWebSecurityOrigin::AllowSubdomains );
           }
         }
