diff --git a/config.cc b/config.cc
index 019e24f..40ea062 100644
--- a/config.cc
+++ b/config.cc
@@ -323,6 +323,7 @@ Programs makeDefaultPrograms()
   programs.push_back( Program( false, Program::Audio, "428b4c2b905ef568a43d9a16f59559b0", "Festival", "festival --tts", "" ) );
   programs.push_back( Program( false, Program::Audio, "2cf8b3a60f27e1ac812de0b57c148340", "Espeak", "espeak %GDWORD%", "" ) );
   programs.push_back( Program( false, Program::Html, "4f898f7582596cea518c6b0bfdceb8b3", "Manpages", "man -a --html=/bin/cat %GDWORD%", "" ) );
+  programs.push_back( Program( false, Program::PlainText, "4cfed3f5b18b036b78c0390e73c7c787", "Google Translate", "trans -e google -s en -t zh -show-original y -show-original-phonetics n -show-translation y -no-ansi -show-translation-phonetics n -show-prompt-message n -show-languages y -show-original-dictionary n -show-dictionary n -show-alternatives n &quot;%GDWORD%&quot;", "" ) );
 #endif
 
   return programs;
